caom2-search-server/src/main/java/ca/nrc/cadc/tap/TAPServlet.java

```diff
@@ -224,7 +229,18 @@ RegistryClient getRegistryClient() {
         return new RegistryClient();
     }
 
-    private URI lookupServiceURI() {
-        return getServiceID(TAP_SERVICE_URI_PROPERTY_KEY, DEFAULT_TAP_SERVICE_URI);
+    private URI lookupServiceURI(final HttpServletRequest request) {
+        // Create the TAP job to prepare to be executed.
+        // Check to see if this is an MAQ job
+        URI tapServiceURI = DEFAULT_TAP_SERVICE_URI;
+        String tapServiceKey = TAP_SERVICE_URI_PROPERTY_KEY;
+        String useAlt = request.getParameter("USEMAQ");
+        if ((request.getParameter("USEMAQ") != null)
+            && (request.getParameter("USEMAQ").equals("true")) ) {
```

Comment:
The `useAlt` variable can be re-used here.  Also please use the `StringUtil.hasText()` to check strings.